distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -1754,3 +1754,8 @@ adapt_unified_column column expected_type =
    A shorthand to be able to share the implementations between in-memory and
    database.
 simple_unary_op column op_kind = column.make_unary_op op_kind
+
+## PRIVATE
+Materialized_Column.from (that:Column) =
+    _ = [that]
+    Error.throw (Illegal_Argument.Error "Currently cross-backend operations are not supported. Materialize the column using `.read` before mixing it with an in-memory Table.")
```

Comment:
Wow this is such a clever use of the conversions! ❤️  I didn't think we could use them that way but that makes the error handling code much cleaner and handles all of our methods at once.

Kudos!